### ISSUE TITLE
Double tap to Zoom

### ISSUE THREAD

**Issue by pedroemonteiro** [kollerlukas/Camera-Roll-Android-App#68]
When double tapping to zoom don't go straight to full resolution, go first to "Fit Screen".

Great app.

**Comment by dkanada** [comment]
Just in case you weren't aware, double tapping and then dragging your finger up and down on the second tap is a granular zoom control. I didn't even know it existed until about a year ago.

**Comment by pedroemonteiro** [comment]
@dkanada Very interesting behavior, didn't know about it. Still nothing to do with what I meant with this feature request.

**Comment by dkanada** [comment]
Yeah, I just thought it might be a decent alternative for now.

**Comment by Wiredframe** [comment]
It would be nice if there was an option to open an item in fullscreen (without file name and toolbar at the bottom). I really love this gallery!

**Comment by fredgolightly** [comment]
I love Quickpic's zoom steps - that is, fit to screen when opening image, then 1st double tap = fit to edges, 2nd double tap = 100%. 

I find it strange that CameraRoll instantly goes in to what looks like 200%, as the image then looks horrible as it's zooming past what's really 'there'. Could you make it not go over 100% initially please?

**Comment by ammodytes** [comment]
Yes, please, can you set app to go on 100% image size on doble tap, not more then 100%. Thank you very much.

**Comment by kollerlukas** [comment]
https://github.com/kollerlukas/Camera-Roll-Android-App/commit/a8a0710fcfc9ac5fda14357d34185624c5ad80d4
